### PR TITLE
Added a bug for Password Encryption

### PR DESCRIPTION
**Pull request by gautii24**
I have reported a bug for password encryption

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,35 +1,19 @@
...
-4. See error
+2. Assign the value to parameter and select generate and check if it is encrypted
 
-**Expected behavior**
-A clear and concise description of what you expected to happen.
+3. Now run the business flow
 
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
+4. After completion of the execution , click on generate reports and go to the destination where results are stored.
```
Comment (ambarkhan) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4796878)

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 24
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4796878&bid=15966423)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,35 +1,19 @@
...
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+2. Assign the value to parameter and select generate and check if it is encrypted
 
-**Expected behavior**
-A clear and concise description of what you expected to happen.
+3. Now run the business flow
```
Comment (ambarkhan) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4796878)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -1,35 +1,19 @@
...
-about: Create a report to help us improve
+Bug : 
+Password encryption works fine during execution, but after generating the report it gets displayed without encryption.
 
----
+To Reproduce Steps to reproduce the behavior:
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+1. Go to add parameters and select Parameter Password String
```
Comment (ambarkhan) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4796878)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -1,35 +1,19 @@
...
 
-**To Reproduce**
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+2. Assign the value to parameter and select generate and check if it is encrypted
```
Comment (ambarkhan) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4796878)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -1,35 +1,19 @@
...
 
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
+4. After completion of the execution , click on generate reports and go to the destination where results are stored.
 
-**Desktop (please complete the following information):**
- - OS: [e.g. iOS]
- - Browser [e.g. chrome, safari]
- - Version [e.g. 22]
+5. In execution details check for the details
```
Comment (ambarkhan) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4796878)

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 24
- Added 6
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4796878&bid=15966423)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -1,35 +1,14 @@
...
-
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
+To Reproduce Steps to reproduce the behavior:
 
-**Desktop (please complete the following information):**
- - OS: [e.g. iOS]
- - Browser [e.g. chrome, safari]
- - Version [e.g. 22]
+1. Go to add parameters and select Parameter Password String
```
Comment (ambarkhan) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4796878)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 7]
```diff
@@ -1,35 +1,14 @@
...
+2. Assign the value to parameter and select generate and check if it is encrypted
+3. Now run the business flow
+4. After completion of the execution , click on generate reports and go to the destination where results are stored.
```
Comment (ambarkhan) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4796878)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 8]
```diff
@@ -1,35 +1,14 @@
...
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
+To Reproduce Steps to reproduce the behavior:
 
-**Desktop (please complete the following information):**
- - OS: [e.g. iOS]
- - Browser [e.g. chrome, safari]
- - Version [e.g. 22]
+1. Go to add parameters and select Parameter Password String
+2. Assign the value to parameter and select generate and check if it is encrypted
```
Comment (ambarkhan) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4796878)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 9]
```diff
@@ -1,35 +1,14 @@
...
+3. Now run the business flow
+4. After completion of the execution , click on generate reports and go to the destination where results are stored.
+5. In execution details check for the details
 
-**Smartphone (please complete the following information):**
- - Device: [e.g. iPhone6]
- - OS: [e.g. iOS8.1]
- - Browser [e.g. stock browser, safari]
- - Version [e.g. 22]
+Expected behavior : Password in the report should be displayed in encrypted form
```
Comment (ambarkhan) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-consecutive-blank-lines] Remove 1 line after node](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4796878)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 10]
```diff
@@ -1,35 +1,14 @@
...
 
-**Desktop (please complete the following information):**
- - OS: [e.g. iOS]
- - Browser [e.g. chrome, safari]
- - Version [e.g. 22]
+1. Go to add parameters and select Parameter Password String
+2. Assign the value to parameter and select generate and check if it is encrypted
+3. Now run the business flow
+4. After completion of the execution , click on generate reports and go to the destination where results are stored.
+5. In execution details check for the details
```
Comment (ambarkhan) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4796878)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 11]
```diff
@@ -1,35 +1,14 @@
...
-If applicable, add screenshots to help explain your problem.
+To Reproduce Steps to reproduce the behavior:
 
-**Desktop (please complete the following information):**
- - OS: [e.g. iOS]
- - Browser [e.g. chrome, safari]
- - Version [e.g. 22]
+1. Go to add parameters and select Parameter Password String
+2. Assign the value to parameter and select generate and check if it is encrypted
+3. Now run the business flow
```
Comment (ambarkhan) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4796878)

**Comment by JinendraGhodke** [comment]
Please report it under issues section and not under PR

**Comment by JinendraGhodke** [comment]
Please report it under issues section and not under PR

**Pull request closed without merging**